client/perffarm-client.py

```diff
@@ -19,8 +19,10 @@
 
 from settings_local import *
 from settings import *
-API_URL = 'http://127.0.0.1:8000/'
-MACHINE_SECRET = '610f79063e62e6ad09460ac2c4e66da0386dc89b'
+# API_URL = 'http://127.0.0.1:8000/'
+API_URL = 'http://127.0.0.1:8000/upload/'
+#MACHINE_SECRET = '610f79063e62e6ad09460ac2c4e66da0386dc89b'
+MACHINE_SECRET = 'e984c3017cd1a0dff0ef9f0c394a5c285e421411'
```

Comment:
pls remove the commented lines. 
This machine secrete is specific to your environment you don't need to checkin